test/org/sagebionetworks/bridge/services/AuthenticationServiceImplTest.java

```diff
@@ -154,7 +154,18 @@ public void canResendEmailVerification() throws Exception {
         try {
             Email email = new Email(testUser.getStudyIdentifier(), user.getEmail());
             authService.resendEmailVerification(user.getStudyIdentifier(), email);
-        } catch (ConsentRequiredException e) {
+        } finally {
+            helper.deleteUser(user);
+        }
+    }
+    
+    @Test
+    public void resendEmailVerificationLooksSuccessfulWhenNoAccount() throws Exception {
```

Comment:
No I mean should we get a unit test for requestResetPassword for when there's no account?